api/v1/errors/RedisError.js

```diff
@@ -0,0 +1,19 @@
+var UnprocessableRequestError = require('./UnprocessableRequestError');
```

Comment:
Make this a sub-class of the `APIError` instead. An `UnprocessableRequestError` implies that there must be a request associated with what happened